src/js_native_api_v8.cc

```diff
@@ -893,6 +893,28 @@ napi_status napi_get_property_names(napi_env env,
   return GET_RETURN_STATUS(env);
 }
 
+NAPI_EXTERN napi_status napi_get_own_property_names(napi_env env,
+                                                    napi_value object,
+                                                    napi_value* result) {
+  NAPI_PREAMBLE(env);
+  CHECK_ARG(env, result);
+
+  v8::Local<v8::Context> context = env->context();
+  v8::Local<v8::Object> obj;
+  CHECK_TO_OBJECT(env, context, obj, object);
+
+  v8::MaybeLocal<v8::Array> maybe_own_propertynames = obj->GetOwnPropertyNames(
+      context,
+      static_cast<v8::PropertyFilter>(v8::PropertyFilter::SKIP_SYMBOLS),
```

Comment:
Is there a reason we don't or in the flag `v8::PropertyFilter::NON_ENUMERABLE` like we do in `napi_get_own_property_names()`?

Reply:
I think we follow the [MDN doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames) is better which the returns included NON-ENUMERABLE and ENUMERABLE :)